install-preq.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -ex
+
+# yum update -y
+yum install -y centos-release-scl scl-utils epel-release
+yum install -y git wget gnupg2 ccache
+yum install -y python-pip
+yum install -y devtoolset-8 devtoolset-7
+# pip install --upgrade pip
+# pip install --upgrade virtualenv
+scl enable devtoolset-8 bash
+scl enable devtoolset-7 bash
```

Comment:
all these packages should be installed already by the `install-deps.sh`, can you expand on why you needed to install these?

Reply:
It happend that these images were not installed and errored out due to their unavailabilty. So i installed them explicitly

Reply:
so then we need to fix `install-deps.sh`. That script is taken from the upstream Ceph repo, so we need to figure out the root cause, otherwise we won't be able to explain any subsequent failure

Reply:
yeah. so please take a look at [this](https://github.com/JayjeetAtGithub/ceph-builder/blob/centos-base-nautilus/install-preq.sh). Removing the devtoolsets, the image build again starts failing [here](https://travis-ci.com/github/JayjeetAtGithub/ceph-builder)

Reply:
what I meant is that this script as a whole shouldn't be needed.

Reply:
Oh I see. Yeah thats the ideal. So, what should be done now ? Should we try to fix in the upstream or continue with patches like this ?

Reply:
we need to remove this script. Building the image should work without it. If there's a problem when building the ceph-builder image after we remove this script, then it'll very likely be something on our end. The upstream script is used extensively by the ceph community, and has some minor tweaks for skyhook (install arrow/parquet). It could still be something on the upstream script, but it's unlikely